static/formats/s3csv/supply/item_category_ifrc_standard.xsl

```diff
@@ -20,8 +20,7 @@
             supply_catalog...............
             supply_item_category.........
 
-        @todo:
-            - remove org_organisation (?)

```

Comment:
That has a ? after it.  So should find out if that means it's not certain this is wanted.  Also need to find out if this goes along with some other changes, e.g. in the supply module.